pkgs/test/default.nix

```diff
@@ -51,6 +51,8 @@ with pkgs;
 
   php = recurseIntoAttrs (callPackages ./php {});
 
+  ripunzip = callPackages ../build-support/fetchzip/tests.nix { unzip = ripunzip; };
```

Comment:
This is not a "core" functionality, it should not here. I'm going to rework this.